tests/automation_framework/src/libs/test_base.py

```diff
@@ -242,58 +243,97 @@ def post_json_msg(self, request_file):
         logger.info('**********Received Response*********\n%s\n', response)
         return response
 
-    def run_test(self, config_file):
-        test_name = inspect.stack()[1].function
-        test_data = read_config(config_file, test_name)
-        result_response = ""
+    def avalon_uncomputed(self, request_file=None, test_data=None):
```

Comment:
Do you need this request_file parameter and the else part as we would now always send the test_data? I feel the function should just have the json.dumps and then the postmsg

Reply:
fixed